digit-health-services/health-delivery-service/src/main/resources/application-test.properties

```diff
@@ -0,0 +1,43 @@
+# DATASOURCE CONFIGURATION
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.jpa.hibernate.ddl-auto=create
+
+
+# FLYWAY CONFIGURATION
+spring.flyway.url=jdbc:h2:mem:testdb
+spring.flyway.user=sa
+spring.flyway.password=password
+spring.flyway.table=public
+spring.flyway.baseline-on-migrate=true
+spring.flyway.outOfOrder=true
+spring.flyway.locations=classpath:/db/migration/main
+spring.flyway.enabled=true
+
+
+# KAFKA SERVER CONFIGURATIONS
+kafka.config.bootstrap_server_config=localhost:9092
+spring.kafka.consumer.value-deserializer=org.egov.tracer.kafka.deserializer.HashMapDeserializer
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.group-id=health-sync-service
+spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
+spring.kafka.consumer.properties.spring.json.use.type.headers=false
+
+# KAFKA CONSUMER CONFIGURATIONS
+kafka.consumer.config.auto_commit=true
+kafka.consumer.config.auto_commit_interval=100
+kafka.consumer.config.session_timeout=15000
+kafka.consumer.config.auto_offset_reset=earliest
+# KAFKA PRODUCER CONFIGURATIONS
+kafka.producer.config.retries_config=0
+kafka.producer.config.batch_size_config=16384
+kafka.producer.config.linger_ms_config=1
+kafka.producer.config.buffer_memory_config=33554432
+#org.egov.detailed.tracing.enabled = false
+
+# FILE STORE SERVICE
+egov.filestore.host=http://localhost:8083
```

Comment:
We can remove unused env variables like filestore service which we may not use in this service

Reply:
updated.